doc/api/util.md

```diff
@@ -1429,6 +1429,8 @@ changes:
   * `allowPositionals` {boolean} Whether this command accepts positional
     arguments.
     **Default:** `false` if `strict` is `true`, otherwise `true`.
+  * `allowNegative` {boolean} Whether allow negative options.
```

Comment:
```suggestion
  * `allowNegative` {boolean} Whether allow negative options prefixed with `--no-`.
```

Reply:
"Negative options" is a bit unclear. Maybe something more explicit?

```suggestion
  * `allowNegative` {boolean} If `true`, allows explicitly setting boolean options to `false` by prefixing the option name with `--no-`.
```

Reply:
Resolved. Thanks